File: app/src/main/java/com/neome/api/meta/base/dto/StudioGrid.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdGrid;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioGrid extends StudioComposite
{
  @Nullable
  public StudioMapOfLayoutGrid layoutGridMap;

  @Nullable
  public Long maxRows;

  @Nullable
  public MetaIdVar maxRowsVarId;

  public MetaIdGrid metaId;

  @Nullable
  public Long minRows;

  @Nullable
  public MetaIdVar minRowsVarId;

  @Nullable
  public StudioMapOfActionPermission rowActionPermissionMap;

  @Nullable
  public MetaIdField showAllRowsFieldId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioGrid.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdGrid
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioComposite
import com.neome.api.meta.base.dto.StudioMapOfActionPermission
import com.neome.api.meta.base.dto.StudioMapOfLayoutGrid

interface StudioGrid : StudioComposite
{
  val layoutGridMap: StudioMapOfLayoutGrid?
  val maxRows: Long?
  val maxRowsVarId: MetaIdVar?
  val metaId: MetaIdGrid
  val minRows: Long?
  val minRowsVarId: MetaIdVar?
  val rowActionPermissionMap: StudioMapOfActionPermission?
  val showAllRowsFieldId: MetaIdField?
}